var/spack/repos/builtin/packages/grid/package.py

```diff
@@ -0,0 +1,149 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+# ----------------------------------------------------------------------------
+# If you submit this package back to Spack as a pull request,
+# please first remove this boilerplate and all FIXME comments.
+#
+# This is a template package file for Spack.  We've put "FIXME"
+# next to all the things you'll want to change. Once you've handled
+# them, you can save this file and test your package like this:
+#
+#     spack install grid
+#
+# You can edit this file again by typing:
+#
+#     spack edit grid
+#
+# See the Spack documentation for more information on packaging.
+# ----------------------------------------------------------------------------
+
+from spack.package import *
+
+
+class Grid(AutotoolsPackage):
+    """Data parallel C++ mathematical object library."""
+
+    homepage = "https://github.com/paboyle/Grid"
+    url      = "https://github.com/paboyle/Grid/archive/refs/tags/0.8.2.tar.gz"
+    git      = "https://github.com/paboyle/Grid.git"
+
+    version('develop', branch='develop')
+    # v0.8.2 is currently the latest "numbered" version, but it doesn't build
+    # because during bootstrap it automatically downloads Eigen, but it fetches
+    # it from the old Bitbucket repository which doesn't exist anymore.
+    version('0.8.2', sha256='2b3389a75dec78f154aa28c8828d81a251b0331284f37cede95747255adef13a', deprecated=True)
```

Comment:
Can we avoid adding a version to a package that is deprecated since the beginning?

Reply:
Fair enough.  I just started with that for `spack create`, so I left it there, but yeah, it doesn't work at the moment unless someone motivated (i.e. not me) patches out the URL